### PR TITLE
[bitnami/grafana-operator] extraVolumes, extraVolumeMounts, sidecars

### PR DESCRIPTION
**Pull request by joshuasimon-taulia**
**Description of the change**

Add additional extraVolumes & extraVolumeMounts parameters to the grafana deployment. one example use case is running a cloudsql-proxy sidecar

**Benefits**

Users can configure their own volumes.

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

mostly copied from the bitnami haproxy chart

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/grafana-operator/values.yaml [hunk 1]
```diff
@@ -545,6 +545,23 @@ grafana:
     timeoutSeconds: 5
     failureThreshold: 6
     successThreshold: 1
+  ## @param extraVolumes Optionally specify extra list of additional volumes for the grafana pod(s)
+  ##
+  extraVolumes: []
+  ## @param extraVolumeMounts Optionally specify extra list of additional volumeMounts for the grafana container(s)
+  ##
+  extraVolumeMounts: []
+  ## @param sidecars Add additional sidecar containers to the grafana pod(s)
```
Comment (migruiz4) [inline bitnami/grafana-operator/values.yaml:554]: Could you please add this small change?
```suggestion
  ## @param grafana.extraVolumes Optionally specify extra list of additional volumes for the grafana pod(s)
  ##
  extraVolumes: []
  ## @param grafana.extraVolumeMounts Optionally specify extra list of additional volumeMounts for the grafana container(s)
  ##
  extraVolumeMounts: []
  ## @param grafana.sidecars Add additional sidecar containers to the grafana pod(s)
```

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Hi @joshuasimon-taulia,

Thank you very much for your contribution!

Your PR looks good to me, but I left some comments requesting minor changes.  Could you please have a look at them?

**Comment by joshuasimon-taulia** [comment]
done

**Review by migruiz4 — APPROVED** [review]
Thanks @joshuasimon-taulia, looks good to me!

**Pull request merged**